docs/patterns/valet-key.md

```diff
@@ -136,12 +143,16 @@ public class ValuesController : ApiController
       };
       blobSasBuilder.SetPermissions(BlobSasPermissions.Write);
   
+      var storageSharedKeyCredential = new StorageSharedKeyCredential(blobServiceClient.AccountName, key.Value);
+      var sas = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString();
+
       return new StorageEntitySas
       {
-          BlobUri = blob.Uri,
-          Credentials = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString()
+          BlobUri = blobClient.Uri,
+          Credentials = sas
```

Comment:
```suggestion
          Credentials = blobSasBuilder.ToSasQueryParameters(storageSharedKeyCredential).ToString()
```

And then remove line 147.  It's okay if it's a bit different than the impl.  Effectively the same.